cuda:0
config5 2channel_indoor_v1
Traceback (most recent call last):
  File "/home/ericji3/ECE496/ECE496/src/train_model.py", line 118, in <module>
    main()
  File "/home/ericji3/ECE496/ECE496/src/train_model.py", line 28, in main
    train_loader, val_loader, test_loader = make_dataset.import_indoor_data()
  File "/home/ericji3/ECE496/ECE496/src/data/make_dataset.py", line 184, in import_indoor_data
    train_dataset0 = ImageFolder(root='../../../../../../data/amitabh3/bedroom_193k_prequalified/train', classes = ['real', 'gen'], transform = transform)
TypeError: __init__() got an unexpected keyword argument 'classes'